src/main/java/uk/gov/companieshouse/itemgroupconsumer/ApiClientService.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.companieshouse.itemgroupconsumer;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.InternalApiClient;
+import uk.gov.companieshouse.sdk.manager.ApiSdkManager;
+
+@Component
+public class ApiClientService {
+
+    private final String internalApiUrl;
+
+    public ApiClientService(@Value("${internal.api.url}") String internalApiUrl) {
+        this.internalApiUrl = internalApiUrl;
+    }
+
+    public InternalApiClient getInternalApiClient() {
+        final var client = ApiSdkManager.getPrivateSDK();
+        client.setInternalBasePath(internalApiUrl);
```

Comment:
This may not be necessary?

Reply:
Can you be more specific? What isn't necessary? Why?

Reply:
We seem to get away with not configuring it in examples such as this one?: https://github.com/companieshouse/digital-certified-copy-processor/blob/7dae836559afdd7f828151f1f428c17141428e50/src/main/java/uk/gov/companieshouse/digitalcertifiedcopyprocessor/service/ApiClientService.java#L15-L17.